aiida_aurora/utils/cycling_analysis.py

```diff
@@ -357,8 +356,29 @@ def add_analysis(data: dict) -> str:
 
     Returns
     -------
-    `str`
+    `Styler | str`
         The details of the analysis.
     """
-    # TODO replace str(data) with something insightful, and clean!
-    return str(data) if data else "ERROR! Failed to find or parse output"
+
+    if data:
+
+        COLUMNS = {
+            "time": "Time (s)",
+            "I": "I (A)",
+            "Ewe": "Ewe (V)",
+        }
```

Comment:
Just wondering (I do not remember) whether the units of `data` are always these ones (s, A, V).

Reply:
When using the AiiDA-Aurora workflow, as far as I know, yes. When running via EC-lab, for example, current may be in mA, in which case the new raw-results importing mechanics (see epfl-theos/aiidalab-aurora#60) would need to start from pre-processed JSON files that converted current to A. This is understood already from experience and will be added to the documentation.